src/lexedata/importer/interleaved.py

```diff
@@ -9,13 +9,19 @@
 import re
 import csv
 import sys
+import logging
+from pathlib import Path
 
 import openpyxl
 
-if __name__ == "__main__":
+# TODO: move this logger part after the argument parser (after merge)
+logger = logging.getLogger(__file__)
```

Comment:
Have you seen the `lexedata.cli.logger` that I have put in several places now?

Reply:
Yes, that's what the TODO is for